### PR TITLE
ignoreDevicePixelRatio :: option for ignoring window.devicePixelRatio

### PR DESCRIPTION
**Pull request by biosonic**
On mobile devices I was unable to manipulate crop-area due to mismatch of crop-area touch/click regions and crop-area display.

### CODE REVIEW

**Comment by WebSeed** [comment]
Hey sorry, just seen this - will take a look later

File: README.md [hunk 1]
```diff
@@ -59,6 +59,7 @@ var crop = Crop.create({
     x: 100,
     y: 500
   },
+  ignoreDevicePixelRatio: false,
```
Comment (WebSeed) [inline README.md]: Good idea, but probably better to accept explicit `devicePixelRatio` value i.e. in your case, just set `devicePixelRatio: 1` - does that sound okay?

**Review by willbamford — CHANGES_REQUESTED** [review]
Might be better to just have `devicePixelRatio` passed as param (taking a number) instead of `ignoreDevicePixelRatio`

File: README.md (hunk 1 above)
Comment (biosonic) [reply README.md]: I've tried that and it works, but I was afraid that changing _devicePixelRatio_ could make problem somewhere else in app.

I was making some crop area with Vue and using _ignoreDevicePixelRatio_ it's responsive now :). 

Very nice interface, thank you.

**Pull request closed without merging**